Screens/QudUX_GameStatsScreen.cs

```diff
@@ -0,0 +1,256 @@
+		public List<EnhancedScoreEntry> ScoreList;
+		private enum StatsScreen
+		{
+			GamesList = 1,
+			LevelStats,
+			DeathStats,
+		} 
+
+		private StatsScreen currentScreen;
+		public void Init(TextConsole console, ScreenBuffer buffer)
+		{
+			Console = console;
+			Buffer = buffer;
+		}
+		public ScreenReturn Show(GameObject GO)
+        {
+            currentScreen = StatsScreen.GamesList;
+
+            //Logger.Log("Loading scoreboard");
+            Table scoreTable, levelsTable, deathCauseTable;
+			bool showAbandonned = true;
+
+            FillTables(out scoreTable, out levelsTable, out deathCauseTable, showAbandonned );
+
+
+            while (true)
+            {
+                DisplayScreenFrame();
+                Buffer.Goto(55, 0);
+                Buffer.Write(((int)currentScreen).ToString() + "/3");
+
+                Table currentTable = null;
+                if (currentScreen == StatsScreen.GamesList)
+                {
+                    scoreTable.Display(Buffer, 1, 1);
+                    currentTable = scoreTable;
+                }
+                else if (currentScreen == StatsScreen.LevelStats)
+                {
+                    levelsTable.Display(Buffer, 1, 1);
+                    currentTable = levelsTable;
+
+                }
+                else if (currentScreen == StatsScreen.DeathStats)
+                {
+                    deathCauseTable.Display(Buffer, 1, 1);
+                    currentTable = deathCauseTable;
+                }
+
+                Buffer.Goto(2, 23);
+                Buffer.Write(ScoreList.Count.ToString() + " {{O|games}}");
+                /*
+				// debug table
+				Buffer.Goto(1,24);
+				Buffer.Write($"si:{scoreTable.SelectedIndex} of:{scoreTable.Offset} mvr:{scoreTable.MaxVisibleRows} cvr:{scoreTable.CurrentVisibleRows} ");
+				*/
+
+                Console.DrawBuffer(Buffer);
+
+                Keys keys = Keyboard.getvk(Options.MapDirectionsToKeypad);
+
+                if (keys == Keys.Escape || keys == Keys.NumPad5)
+                {
+                    GameManager.Instance.PopGameView();
+                    return ScreenReturn.Exit;
+                }
+
+				if (keys == Keys.OemQuestion)
+				{
+					ShowQuickKeys();
+				}
+
+                if (currentTable != null)
+                {
+                    if (keys == Keys.NumPad3 || keys == Keys.Next)
+                    {
+                        currentTable.ScrollPage(1);
+                    }
+
+                    if (keys == Keys.NumPad9 || keys == Keys.Prior)
+                    {
+                        currentTable.ScrollPage(-1);
+                    }
+
+                    if (keys == Keys.NumPad2)
+                    {
+                        currentTable.MoveSelection(1);
+                    }
+
+                    if (keys == Keys.NumPad8)
+                    {
+                        currentTable.MoveSelection(-1);
+                    }
+
+					if (keys == (Keys)131137)
```

Comment:
Replace number with Keys enum LogicalOr
```suggestion
					if (keys == (Keys.Control | Keys.A))
```